charts/celeborn/values.yaml

```diff
@@ -53,28 +53,28 @@ volumes:
     - mountPath: /mnt/celeborn_ratis
       hostPath: /mnt/celeborn_ratis
       type: hostPath
-      size: 1Gi
+      capacity: 1Gi
```

Comment:
I'm not sure, but 1GiB might be too small?

Reply:
We can plus this to 100 Gi, WDYT?

Reply:
100Gi is good to me. IMO it's better to let users decrease if needed, rather than throw exception because of low capacity when there are actually enough space.

Reply:
Yeah, after this pr, user can modify celeborn workers disk capacity in `values.yml`.